inference/wav2letter/evaluate_inference.py

```diff
@@ -0,0 +1,166 @@
+import pandas as pd
+
+from jiwer import wer
+from pathlib import Path
+from wav2letter import inference
+
+
+def print_result(result):
+    print(" ".join([w.word for w in result.words]))
+
+
+model = inference.load_model(
+    input_files_base_path="/local/working/dir/2020_12_01_v1_tds_ctc_serialized"
+)
+
+chunk_size = 16000  # 32000 = 1 sec (mobile sends chunks of 3200)
+
+local_dir = "/home/tetianamyronivska/test_segments_by_provider"
```

Comment:
Is this run in the container?

Reply:
yeah, but we can update the path easily.